frontend-opal/components/Header2/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {
+  Button,
+  View,
+  TextInput,
+  Image,
+  Text,
+} from 'react-native';
+import styles from './styles.js';
+
+const Header = () => {
+  const narTitles = [
+    { key: 'PORTFOLI' },
+    { key: 'WATCHLIST' },
+  ];
+  const renderItems = (titles) => (
+    titles.map((title) => (
+      <View style={styles.narTitle} key={title.key}>
+        <Button title={title.key} color="#e17055" />
+      </View>
+
+    ))
+  );
+  return (
+    <View>
+      <View style={styles.search}>
```

Comment:
@martinupset use the `Header` component https://docs.nativebase.io/Components.html#header-icon-headref

Reply:
ok